testData/psi/shell/conditionalCommand.bash

```diff
@@ -0,0 +1 @@
+<caret>[ -z "$var" -a -n "$var2" ]
```

Comment:
Could you add another test file with the `<caret>` on the closing bracket to make sure that the inspection is enabled there as well?